src/features/modals/CreateProjectModal.tsx

```diff
@@ -1,59 +1,57 @@
-import { signal } from "@preact/signals-core";
-import { type Ref, useRef, useState } from "preact/hooks";
+import { useRef, useState } from "preact/hooks";
 import { PlusIcon } from "../icons/PlusIcon";
 import { create, useProjectsList } from "../../lib/projects";
+import { Modal } from "./Modal";
```

Comment:
I abstracted all the modal styling out of this component for less redundancy and to make it easier to keep the modal UX consistent.